### PR TITLE
Huffman Decoding in Python done

### PR DESCRIPTION
**Pull request by Riyazul555**
Check this out...closes Huffman Decoding in Python #31 
Thanks

### CODE REVIEW

File: Huffman Decoding/HuffmanDecoding.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        cod[nod] = val
+        return
+    dfs(G[nod][0], val + "0", G, cod)
+    dfs(G[nod][1], val + "1", G, cod)
+
+def main(argv):
+    fin = open(argv[1], 'r')
+    fout = open(argv[2], 'w')
+    n = int(fin.readline())
+    fr = [0] * nMax
```
Comment (iuliagroza) [inline Huffman Decoding/HuffmanDecoding.py:20]: Where is nMax defined?

File: Huffman Decoding/HuffmanDecoding.py
Comment (iuliagroza) [inline Huffman Decoding/HuffmanDecoding.py:1]: Change name to "Huffman Decoding/huffman_decoding.py", as Python is snake_case.

File: Huffman Decoding/HuffmanDecoding.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    val = [0] * nMax
```
Comment (iuliagroza) [inline Huffman Decoding/HuffmanDecoding.py:21]: ditto

**Review by iuliagroza — COMMENTED** [review]
Can we link the PRs to their associated issue? It's much easier to update their status that way.

**Comment by Riyazul555** [comment]
Done @iuliagroza

**Comment by Riyazul555** [comment]
Fixed @iuliagroza

**Comment by iuliagroza** [comment]
> Fixed @iuliagroza

Are there any new commits?

**Comment by Riyazul555** [comment]
No @iuliagroza 

> > Fixed @iuliagroza
> 
> Are there any new commits?

**Comment by Riyazul555** [comment]
@iuliagroza if everything is fine merge this pR then 
Thanks

**Comment by iuliagroza** [comment]
@Riyazul555 Maybe let's at least rename the file to align with the Python 3 styleguide? Thanks!

**Comment by Riyazul555** [comment]
I guess I already renamed it to Python snakecase style @iuliagroza

**Review by iuliagroza — APPROVED** [review]

**Pull request merged**